tests/benchmarks/test_parquet.py

```diff
@@ -0,0 +1,109 @@
+"""
+Parquet-related benchmarks.
+"""
+import uuid
+
+import dask.dataframe as dd
+import dask.datasets
+import distributed
+import fsspec
+import pandas
+import pytest
+from coiled.v2 import Cluster
+
+
+@pytest.fixture(scope="module")
+def parquet_cluster():
+    with Cluster(
+        f"parquet-{uuid.uuid4().hex[:8]}",
+        n_workers=15,
+        worker_vm_types=["m5.xlarge"],
+        scheduler_vm_types=["m5.xlarge"],
+    ) as cluster:
+        yield cluster
+
+
+@pytest.fixture(scope="module")
+def parquet_client(parquet_cluster):
+    with distributed.Client(parquet_cluster) as client:
+        client.restart()
```

Comment:
V2 does a decent job of waiting for most workers to arrive by default

Reply:
Heads up that with latest dask/distributed `client.restart()` can cause you to lose workers (and unless you call scale you won't get them back). @jrbourbeau can supply more details.

Reply:
Thanks @ntabris. This should be good now that there's a fix in `distributed`

> V2 does a decent job of waiting for most workers to arrive by default

Nice, that's good to know. Though since we're reusing the cluster I think it would still be good to include `scale` + `wait_for_workers` lines similar to 

https://github.com/coiled/coiled-runtime/blob/c2cb934cca95e7f463cf4daff9fd84798fe1fb82/conftest.py#L91-L97

in case a worker goes down for some reason in an earlier test, we make sure to get back up to the expected number of workers in subsequent tests. Thoughts?

Reply:
Sure, not a problem